php/post-types/class-block-post.php

```diff
@@ -369,6 +396,54 @@ public function save_draft_button() {
 		}
 	}
 
+	/**
+	 * Returns icon fields based on passed classname and icon.
+	 *
+	 * @param string $classname The classname to use for the icon classes.
+	 * @param string $icon      The icon to use as a default.
+	 *
+	 * @return void
+	 */
+	private function get_icons( $classname = 'icon', $icon = '' ) {
```

Comment:
I'm not sure if it's worth spinning this out into its own method. The name confused me – maybe something like `render_icon_input()` would work better?

Reply:
Yeah, I moved it to its own method when I was trying to re-use icons for the block categories. Evidently, you can't create SVGs manually but at the block level. I'll move it back to its own context.

Reply:
I moved this function back to its own context.